3.0.0/README.md

```diff
@@ -140,43 +246,43 @@ Contains a legend to be displayed with the map. Implementations MAY decide to tr
 }
 ```
 
-## 3.8 `maxzoom`
+## 3.12 `maxzoom`
 
 OPTIONAL. Default: 30. >= 0, <= 30.
 
 An integer specifying the maximum zoom level. MUST be >= minzoom.
```

Comment:
Speaking as someone who has found the meanings of maxzoom and minzoom very confusing in the past, I would suggest a more explicit description:

```
OPTIONAL integer. Default: 30. MUST be in range: 0 <= minzoom <= maxzoom <= 30

An integer specifying the maximum zoom level at which tiles are available.
```

Reply:
https://github.com/mapbox/tilejson-spec/pull/36/commits/5e75cf855792a11a4ecaf8d9ae3372dad03ac127